Sources/TuistLoader/Models+ManifestMappers/TestAction+ManifestMapper.swift

```diff
@@ -89,3 +89,26 @@ extension TuistGraph.TestAction {
         )
     }
 }
+
+extension TestPlan {
```

Comment:
Let's be consistent with the other `{Model}+ManifestMapper.swift` and extract this one into a `TestPlan+ManifestMapper.swift` file.